lib/internal/Magento/Framework/App/Test/Unit/Cache/TypeListTest/Type.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Framework\App\Test\Unit\Cache\TypeListTest;
+
+use Magento\Framework\Cache\Frontend\Decorator\TagScope;
+use Magento\Framework\App\Test\Unit\Cache\TypeListTest;
+
+/**
+ * Cache type "Type"
+ */
+class Type extends TagScope
```

Comment:
I am not sure if we need class to only handle constants.
Can we just use private constants in places where they need to be used?